config/deploy.rb

```diff
@@ -1,12 +1,12 @@
 # config/deploy.rb file
 require 'bundler/capistrano'
 
-set :application, "scholarship-application-app"
+set :application, "longshot"
 set :deploy_to, ENV["DEPLOY_PATH"]
 server  ENV["SERVER_NAME"], :app, :web
 
-set :user, "ubuntu"
-set :group, "ubuntu"
+set :user, "dosomething"
```

Comment:
No particular reason other than the other Laravel deployments are using dosomething:dosomething, and I was hoping to standardize these wercker.yml files as much as possible.